### ISSUE TITLE
E94: No matching buffer for ^thesaurus$

### ISSUE THREAD

**Issue by jalvesaq** [beloglazov/vim-online-thesaurus#25]
When I run `:OnlineThesaurusCurrentWord` I get this error message:

```
Error detected while processing function <SNR>19_Lookup:
line    1:
E94: No matching buffer for ^thesaurus$
```

To reproduce the bug, you have to put in your *vimrc*:

```vim
set debug=msg,throw
```

I think that this is a Vim bug because the function `bufwinnr()` should return `-1` without error if the buffer does not exist, but you can avoid it with the following code:

```vim
    if bufexists('^thesaurus$')
        let l:thesaurus_window = bufwinnr('^thesaurus$')
    else
        let l:thesaurus_window = -1
    endif
```

**Comment by av3r4ge** [comment]
Weird.
There is a test that checks whether the value is above -1. Silly me believed the documentation.
Can you open a PR replacing the old `if` with yours? (no need for the local variable in this case either).

**Comment by jalvesaq** [comment]
Sorry, I did see your edits your comment above. Please, ignore my PR and go ahead fixing the issue.

**Comment by av3r4ge** [comment]
I guess you only read the email. I should have probably write the updates in a new comment so you would get them by email too.

I really appreciate you took the time making the PR :+1: 
You can take a look at mine (#26).

**Comment by beloglazov** [comment]
Fixed in #26
